topdown/eval.go

```diff
@@ -617,11 +617,27 @@ func (e *eval) evalCall(terms []*ast.Term, iter unifyIterator) error {
 
 	ref := terms[0].Value.(ast.Ref)
 
+	args := make([]*ast.Term, len(terms)-1)
```

Comment:
This means we'll be plugging the args on every call expressions. For built-in functions it means we'll be plugging twice. Perhaps we should plug the args lazily inside of the resolver...